assets/js/pages/Profile/ProfilePage.test.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+
+import MockAdapter from 'axios-mock-adapter';
+import { toast } from 'react-hot-toast';
+
+import { networkClient } from '@lib/network';
+import { screen, act, render } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { userFactory } from '@lib/test-utils/factories/users';
+
+import ProfilePage from '@pages/Profile';
+
+const axiosMock = new MockAdapter(networkClient);
+const PROFILE_URL = '/api/v1/profile';
+
+jest.mock('react-hot-toast', () => ({
+  toast: {
+    success: jest.fn(),
+    error: jest.fn(),
+  },
+}));
+
+describe('ProfilePage', () => {
+  afterEach(() => {
+    axiosMock.reset();
+    jest.clearAllMocks();
+  });
+
+  it('should show the pre-filled form with profile information', async () => {
+    const user = userFactory.build();
+
+    axiosMock.onGet(PROFILE_URL).reply(200, user);
+
+    await act(async () => {
+      await render(<ProfilePage />);
+    });
+
+    expect(await screen.getByLabelText('email').value).toBe(user.email);
+    expect(await screen.getByLabelText('username').value).toBe(user.username);
+    expect(await screen.getByLabelText('fullname').value).toBe(user.fullname);
+  });
+
+  it('should show the pre-filled form with profile information but disable the form is the user is the default admin', async () => {
+    const user = userFactory.build();
+
+    axiosMock.onGet(PROFILE_URL).reply(200, { ...user, id: 1 });
```

Comment:
You could use `adminUser` factory for this if you want. `adminUser.build()` (always has 1 as ID)

Reply:
done